lib/grape/api.rb

```diff
@@ -64,7 +64,7 @@ def api.inherited(child_api)
       def const_missing(*args)
         if base_instance.const_defined?(*args)
           base_instance.const_get(*args)
-        elsif parent && parent.const_defined?(*args)
+        elsif defined?(parent) && parent.const_defined?(*args)
```

Comment:
Can `parent` ever be `nil` here, cause `defined?(nil)` is `"nil"` and `nil` doesn't have a `const_defined` method.

Reply:
Should this even be `parent` or should it be `@base_parent`? I don't see any definition of `parent` anywhere.

Reply:
Good catch btw :)

Reply:
Not sure, but I think we need some specs here. Try something?